### ISSUE TITLE
Test24 fails under OpenVZ

### ISSUE THREAD

**Issue by martinpopel** [VowpalWabbit/vowpal_wabbit#204]
On openvz-amd64 machine,Test #24 (Actives simulation) gives me slightly different stderr output (and actually a better loss) than expected:

```
RunTests: test 24: stdout OK
 --- diff -N --minimal --side-by-side --suppress-common-lines --ignore-all-space --strip- railing-cr -W 150 train-sets/ref/active-simulation.t24.stderr stderr.tmp
1.005870   1.244963            2       502.2  -1.0000   0.1158      10    |     1.005869   1.244949            2       502.2  -1.0000   0.1158      10
1.007812   1.170191            3       508.2  -1.0000   0.0818       5    |     1.007810   1.170181            3       508.2  -1.0000   0.0817       5
 ...
average loss = 0.240509                                                   |     average loss = 0.235951
best constant = -0.136917                                                 |     best constant = -0.157285
total queries = 889                                                       |     total queries = 884
RunTests: test 24: FAILED: ref(train-sets/ref/active-simulation.t24.stderr) != stderr(stderr.tmp)
```

On 32-bit machines, the test goes OK. On 64-bit machine with uname -r 2.6.32-13-pve, it goes OK as well (even if I use the vw binary compiled on the 2.6.32-5-openvz-amd64 machine).

This is a low-priority issue, no real error. Feel free to close it if no one is interested in finding/fixing the cause.
